math.go

```diff
@@ -104,3 +105,19 @@ func MeanBy[T any, R constraints.Float | constraints.Integer](collection []T, it
 	var sum = SumBy(collection, iteratee)
 	return sum / length
 }
+
+// Round returns the float32/float64 of specified precision
+func Round[T float64 | float32](f T, n ...int) T {
+	var nn = 3
+	if len(n) > 0 {
+		nnn := n[0]
+		if nnn >= 0 {
+			nn = nnn
+			if nn > 15 {
+				nn = 15
+			}
+		}
+	}
+	pow10N := math.Pow10(nn)
+	return T(math.Round(float64(f)*pow10N) / pow10N)
```

Comment:
use this instead:

```go
	integer, fractional := math.Modf(nbr)
	total := integer + math.Round(fractional*pow10N)/pow10N
```

because doing `math.Round(float64(f)*pow10N) / pow10N` might lead to interger overflow

Reply:
up?